app/frontend/registration/main/_campaign_details.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="card-body">
+  <div class="h5 mb-3"><%= t('basics.details') %></div>
+
+  <ul class="list-group">
+
+    <!-- Allocation mode detail -->
+    <% mode_info = get_mode_info(campaign.allocation_mode_before_type_cast) %>
+    <li class="list-group-item d-flex justify-content-between align-items-center">
+      <div>
+        <div class="fw-semibold"><%= t('basics.mode') %></div>
+        <small class="text-muted"><%= mode_info[:mode_name] %></small>
+      </div>
+      <span class="d-inline-flex align-items-center px-3 py-1 rounded-pill small fw-medium <%= mode_info[:badge_class] %>">
+        <%= mode_info[:abbr] %>
+      </span>
+    </li>
+
+    <!-- Capacity detail -->
+    <% total_capacity      = sum_of_nullable(items.map(&:capacity)) %>
+    <% total_capacity_used = items.sum(&:item_capacity_used) %>
+    <% multi_items         = items.size > 1 %>
+
```

Comment:
Try to extract this business logic into the model or into the view helper for campaign.